spec/git_fame_spec.rb

```diff
@@ -88,19 +88,24 @@
   end
 
   describe "#command_line_arguments" do
-    let(:subject) do 
+    let(:subject) do
       GitFame::Base.new({
-        repository: @repository, 
-        exclude: "lib", 
-        bytype: true
-      }) 
+        repository: @repository,
+        exclude: "lib",
+        bytype: true,
+        extensions: "rb,rdoc"
+      })
     end
 
     it "should exclude the lib folder" do
       subject.file_list.include?("lib/gash.rb").should be_false
     end
 
-    let(:author) { subject.authors.first }
+    it "should exclude non rb or rdoc files" do
+      subject.file_list.include?("HISTORY").should be_false
+    end
+
+    let(:author) { subject.authors.find { |author| author.name == "7rans" } }
```

Comment:
This test was breaking because of the authors list wasn't in the same order with the new argument I added. I thought using `find` would be more explicit than using an arbitrary index.